controller/src/test/java/io/pravega/controller/server/SegmentHelperTest.java

```diff
@@ -96,7 +96,7 @@ public void createSegment() {
         @Cleanup
         SegmentHelper helper = new SegmentHelper(factory, new MockHostControllerStore(), executorService());
         CompletableFuture<Void> retVal = helper.createSegment("", "",
-                0, ScalingPolicy.fixed(2), "", Long.MIN_VALUE);
+                0, ScalingPolicy.fixed(2), "", Long.MIN_VALUE, 0L);
```

Comment:
Please put anything but `0L` in these tests and then verify that the value "went through" (i.e, you need to verify it).

Reply:
yeah I think these tests and following tests are not really testing the rolloversize feature because the connection is mocked, rollover size is not passed to the segmentstore. 0 here is just a placeholder to make the exsiting test to pass.

I have added end to end (integration) test to test this feature. In the integration test this parameter is passed down to the segmentstore.